content/kubermatic/master/tutorials_howtos/applications/add-applications-to-cluster/_index.en.md

```diff
@@ -4,4 +4,127 @@ date =  2022-08-03T16:43:41+02:00
+- `Name` -> The name of the Application
+- `Values` -> Value override for Installation. This will be left-merged with the default values of your Installation.
+
+The combination of Namespace and Name must be unique within your cluster.
+
+After you have selected your customizations, the installation-status of your Application can be viewed in the Applications Tab.
+
+{{< figure src="./application_status.png" title="Application Installation Status" >}}
+
+### Creating a New Cluster With Applications
+
+In the cluster creation wizard, you can select applications to install into your cluster.
+KKP will automatically install your selection after the infrastructure is provisioned and the cluster is ready.
+
+Applications can be added in the `Applications` Section of the wizard.
+For a detailed flow and explanation of all customizations see  the ["Adding Applications to an Existing Cluster"](#adding-applications-to-an-existing-cluster) section of this guide.
+
+{{< figure src="./applications_flow_in_cluster_wizard.png" title="Application Section in Cluster Creation Wizard" >}}
+
+Afterwards, you can track the installation-progress in the Applications Tab.
+
+{{< figure src="./application_status.png" title="Application Installation Status" >}}
+
+### Storing Applications in a ClusterTemplate
+
+ApplicationInstallations can also be added to ClusterTemplates in order to re-use them across multiple clusters. In order to do so, select the `Save Cluster Template` option during the Summary step of the cluster creation wizard.
+
+{{< figure src="./save_to_cluster_template.png" title="Saving As Cluster Template" >}}
+
+## Managing Applications via GitOps
+
+KKP Applications are managed via the `ApplicationInstallation` custom Kubernetes resource.
+ApplicationInstallations reside in the user-cluster and represent a desired state of an Application.
+For a full reference of all suported fields, please check the [ApplicationInstallation Reference](#applicationinstallation-reference) Section of this guide.
+
+```yaml
+# Example of an ApplicationInstallation
+apiVersion: apps.kubermatic.k8c.io/v1
+kind: ApplicationInstallation
+metadata:
+  name: prometheus
+  namespace: prometheus
+spec:
+  applicationRef:
+    name: prometheus
+    version: 2.36.2
+  namespace:
+    create: true
+    name: prometheus
+  values:
+    alertmanager:
+      enabled: false
+```
+
+After creating an ApplicationInstallation, you can directly apply it into the desired cluster using kubectl. This will trigger a controller within KKP, which will automatically install your Application. We recommend to apply the ApplicationInstallation to the same namespace as the workload and configuration you want to deploy.
+
+```sh
+kubectl apply -n <namespace> -f <your-appinstall>
+```
+
+You can check the progress of your installation in the `status.helmRelease` field of the ApplicationInstallation you created:
```

Comment:
i would write that in a more general way. especially that if the download of source failed (on first installation) status.helmRelease will be nil.

something like this:
```
You can check the progress of your installation in `status.conditions`. There are 2 conditions:

- ManifestsRetrieved that  indicates if the application's source has been correctly downloaded
- Ready that indicates if the application has been correctly installed or upgraded

eg.
    status:
      conditions:
        ManifestsRetrieved:
          lastHeartbeatTime: "2022-08-09T12:10:59Z"
          message: application's source successfully downloaded
          reason: DownaloadSourceSuccessful
          status: "True"
        Ready:
          lastHeartbeatTime: "2022-08-09T12:11:00Z"
          message: application successfully installed or upgraded
          reason: InstallationSuccessful
          status: "True"


If you use helm as templating method then `status.helmRelease` will contain additional information.
```